spec/models/validations/sales/financial_validations_spec.rb

```diff
@@ -311,42 +309,32 @@
 
     context "when buyer 2 is not a child" do
       before do
-        record.update!(ecstat2: rand(0..8))
-        record.reload
+        record.ecstat2 = rand(0..8)
       end
 
       it "does not add an error if buyer 2 has an income" do
-        record.ecstat2 = rand(0..8)
         record.income2 = 40_000
         financial_validator.validate_child_income(record)
         expect(record.errors).to be_empty
       end
     end
 
     context "when buyer 2 is a child" do
-      it "does not add an error if buyer 2 has no income" do
-        record.saledate = Time.zone.local(2023, 4, 3)
-        record.ecstat2 = 9
-        record.income2 = 0
-        financial_validator.validate_child_income(record)
-        expect(record.errors).to be_empty
-      end
-
-      it "adds errors if buyer 2 has an income" do
-        record.saledate = Time.zone.local(2023, 4, 3)
-        record.ecstat2 = 9
-        record.income2 = 40_000
-        financial_validator.validate_child_income(record)
-        expect(record.errors["ecstat2"]).to include(match I18n.t("validations.financial.income.child_has_income"))
-        expect(record.errors["income2"]).to include(match I18n.t("validations.financial.income.child_has_income"))
-      end
-
-      it "does not add an error if the saledate is before the 23/24 collection window" do
-        record.saledate = Time.zone.local(2022, 4, 3)
-        record.ecstat2 = 9
-        record.income2 = 40_000
-        financial_validator.validate_child_income(record)
-        expect(record.errors).to be_empty
+      context "and saledate is current" do
```

Comment:
does this context need to be here?

Reply:
Ah, not any more because I removed the 2022 test.